decidim-core/config/i18n-tasks.yml

```diff
@@ -85,6 +85,7 @@ search:
 ## Do not consider these keys missing:
 ignore_missing:
 - '{devise,simple_form}.*'
+- errors.messages.taken
```

Comment:
It wouldn't if you do this: https://github.com/AjuntamentdeBarcelona/decidim/pull/597#discussion_r97802015